client/js/navigation/NavigationLayout.js

```diff
@@ -139,6 +139,20 @@ export default createDrawerNavigator(
   },
   {
     contentComponent: props => <DrawerContent {...props} />,
+    contentComponent: props => (
+      <View style={{ flex: 1 }}>
+        <SafeAreaView forceInset={{ top: "always", horizontal: "never" }}>
+          <DrawerItems {...props} />
+          <Button
+            title="Logout"
+            onPress={async () => {
+              await AsyncStorage.clear();
+              props.navigation.navigate("AuthLoading");
```

Comment:
Add error handling for when AsyncStorage or navigation produces an error. Try catch block can fix this.